### DISCUSSION TITLE
Is it possible to customize the response when an error occurs in the API?

### DISCUSSION THREAD

**Discussion by jhh78** [pocketbase/pocketbase#4292] [Q&A]
greetings.

Is it possible to customize respawn data when an error occurs in the API?
for example
Validation error: Code 100
Input error: Code 200
...

Is it possible to customize respawn data in the same way?

**Comment by ganigeorgiev** [comment]
The questions lacks details to understand the use case and what type of customizations or responses you have in mind.

In general, you can overwrite the default error response with the `OnBeforeApiError` hook using [Go](https://pocketbase.io/docs/go-event-hooks/#onbeforeapierror) or [JS](https://pocketbase.io/docs/js-event-hooks/#onbeforeapierror).
